chaosaws/asg/probes.py

```diff
@@ -286,11 +317,14 @@ def get_asg_by_tags(tags: Union[dict, List[Dict[str, str]]],
     page_iterator = client.get_paginator(
         'describe_auto_scaling_groups').paginate(
         PaginationConfig={'PageSize': 100})
+
     asg_descrs = {'AutoScalingGroups': []}
 
     for page in page_iterator:
         asg_descrs['AutoScalingGroups'].extend(page['AutoScalingGroups'])
 
+    print(asg_descrs)
```

Comment:
Oops, probably needs to be removed

Reply:
Ach! Yep, thought I got all the errant prints.